tasks/phantom-dc.rake

```diff
@@ -331,6 +331,20 @@ namespace :'phantom-dc' do
 
     puts "No congressional defaults found for the following members: " + notfound.inspect
   end
+
+  desc "Enable defunct status of congressmember"
+  task :defunct, :bioguide_id, :contact_url do |t, args|
+    cm = CongressMember.find_by!(bioguide_id: args[:bioguide_id])
+    attrs = { defunct: true }
+    attrs.merge!(contact_url: args[:contact_url]) if args[:contact_url]
+    cm.update!(attrs)
+  end
+
+  desc "Disable defunct status of congressmember"
```

Comment:
So they should always have a contact form on their official senate.gov website. That's how they make it into the forms database initially. They become defunct not by the contact form getting taken down but by changes to the HTML structure of the page, or reCAPTCHA, etc.